core-bundle/contao/templates/_new/component/_figure.html.twig

```diff
@@ -54,7 +54,7 @@
 
         {# Caption #}
         {% block caption %}
-            {% if figure.hasMetadata|default(false) and figure.metadata.caption %}
+            {% if figure.metadata and figure.metadata.caption %}
```

Comment:
```suggestion
            {% if figure.metadata.caption %}
```

Would this work as well?

Reply:
I think this would throw an exception because it can boil down to `null.caption`